ortools/java/com/google/ortools/sat/IntVar.java

```diff
@@ -19,11 +19,12 @@
 
 /** An integer variable. */
 public final class IntVar implements Literal, LinearExpr {
-  IntVar(CpModelProto.Builder builder, Domain domain, String name) {
+  IntVar(final CpModelProto.Builder builder, final Domain domain, final String name) {
```

Comment:
since java 8, final is implicit for arguments so you don't need to write it explicitly.

Reply:
True, but there is a difference between effectively final and final. I usually use the latter if possible.
Anyway, you can keep the old version.